tests/functional/src/main/java/org/mule/functional/junit4/ArtifactFunctionalTestCase.java

```diff
@@ -107,6 +107,7 @@
   private static ServiceManager serviceRepository;
   private static ClassLoaderRepository classLoaderRepository;
   private static IsolatedClassLoaderExtensionsManagerConfigurationBuilder extensionsManagerConfigurationBuilder;
+  private static IsolatedClassLoaderExtensionsManagerConfigurationBuilder alternativeExtensionsManagerConfigurationBuilder;
```

Comment:
rename to reloadableXXX and add a jdoc explaining that this is for {@link #mustRegenerateExtensionModels()} to function